components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -109,6 +109,10 @@ public void execute() {
                 GatewayCmdUtils.copyFolder(GatewayCmdUtils.getProjectInterceptorsDirectoryPath(projectName),
                         GatewayCmdUtils.getProjectGenSrcInterceptorsDirectoryPath(projectName));
                 codeGenerator.generate(projectName, true);
+
+                //to indicate the api information which is not used in the code generation process, but included in
+                //definition.yaml
+                MgwDefinitionUtils.FindNotUsedAPIInformation(outStream);
```

Comment:
No need to pass the outStream, lets directly use the System.out in FindNotUsedAPIInformation method. OutStream is also pointing to System.out

Reply:
But in this way, it will have more flexibility to mention its own PrintStream. Shall we proceed in this way, or include System.out.

Reply:
Implemented as suggested.